Mage.Sets/src/mage/cards/c/CyberConversion.java

```diff
@@ -0,0 +1,90 @@
+package mage.cards.c;
+
+import mage.abilities.Ability;
+import mage.abilities.effects.ContinuousEffect;
+import mage.abilities.effects.OneShotEffect;
+import mage.abilities.effects.common.continuous.AddCardTypeTargetEffect;
+import mage.abilities.effects.common.continuous.BecomesFaceDownCreatureAllEffect;
+import mage.abilities.effects.common.continuous.BecomesSubtypeAllEffect;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.cards.ModalDoubleFacedCardHalf;
+import mage.constants.CardType;
+import mage.constants.Duration;
+import mage.constants.Outcome;
+import mage.constants.SubType;
+import mage.filter.common.FilterCreaturePermanent;
+import mage.filter.predicate.Predicate;
+import mage.filter.predicate.Predicates;
+import mage.filter.predicate.permanent.PermanentIdPredicate;
+import mage.game.Game;
+import mage.game.permanent.Permanent;
+import mage.target.Target;
+import mage.target.common.TargetCreaturePermanent;
+import mage.target.targetpointer.FixedTarget;
+
+import java.util.Arrays;
+import java.util.UUID;
+
+/**
+ * @author PurpleCrowbar
+ */
+public final class CyberConversion extends CardImpl {
+
+    public CyberConversion(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[]{CardType.INSTANT}, "{U}{U}");
+
+        // Turn target creature face down. It's a 2/2 Cyberman artifact creature.
+        this.getSpellAbility().addEffect(new CyberConversionEffect());
+        this.getSpellAbility().addTarget(new TargetCreaturePermanent());
+    }
+
+    private CyberConversion(final CyberConversion card) {
+        super(card);
+    }
+
+    @Override
+    public CyberConversion copy() {
+        return new CyberConversion(this);
+    }
+}
+
+class CyberConversionEffect extends OneShotEffect {
+
+    CyberConversionEffect() {
+        super(Outcome.UnboostCreature);
+        this.staticText = "Turn target creature face down. It's a 2/2 Cyberman artifact creature";
+    }
+
+    private CyberConversionEffect(final CyberConversionEffect effect) {
+        super(effect);
+    }
+
+    @Override
+    public CyberConversionEffect copy() {
+        return new CyberConversionEffect(this);
+    }
+
+    @Override
+    public boolean apply(Game game, Ability source) {
+        Predicate<Permanent> pred = new PermanentIdPredicate(UUID.randomUUID());
```

Comment:
this seems like a hack.

Also it looks like the BecomesXxxAllEffects will apply even after the permanent is blinked since no zcc check. And what about Break Open, which should end the effect, right?

Try PermanentReferenceInCollectionPredicate, maybe also with FaceDownPredicate?

The whole thing will be super brittle because it isn't applying on layer 1b as it should, but that's a preexisting issue.